src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -93,12 +93,14 @@ public void checkIndentation() {
         // If the argument list is the first element on the line
         final DetailAST firstChild = getMainAst().getFirstChild();
         final DetailAST parent = getMainAst().getParent();
+
+        int firstChildColumnNo = expandedTabsColumnNo(firstChild);
```

Comment:
I have just extracted the method as a variable and test started failing.
![Screenshot from 2020-09-15 20-23-39](https://user-images.githubusercontent.com/48255244/93226882-709edf00-f791-11ea-85cf-9d7554ec6683.png)

Reply:
It is not same code as it was, because this method `expandedTabsColumnNo(firstChild)` was invoked only when type of parent is not SWITCH_RULE.
So you didn't extract variable, but also changed logic a bit.

Reply:
Damn...Its true. Now I understood. Thanks  a lot. @strkkk.

Reply:
Damn...Its true... Thanks a lot @strkkk